tools/ports/opentelemetry/portfile.cmake

```diff
@@ -0,0 +1,76 @@
+include(vcpkg_common_functions)
```

Comment:
This is very early draft. It is functional. We need to tune it later on by adding the proper support for `feature packages` : build with Abseil, build with Standard Library, and other possible build flavors, like build with prometheus, build with this or with that.